kubernetes/linera-validator/templates/grafana-dashboards-config.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: grafana-dashboards-config
+  labels:
+    grafana_dashboard: "1"
+data:
+  linera-general: {{ .Files.Get "grafana-dashboards/linera-general.json" | quote | indent 4 }}
```

Comment:
Grafana will just get whatever is under data and create a file with that name. So all keys here need to have the `.json` suffix otherwise it creates a file with the wrong extension, which makes Grafana not recognize it when ingesting the dashboards.
This was figured out by trial and error lol

Reply:
But this key does not have the `.json` suffix... right? It's just `linera-general`.

Reply:
Exactly, which was the issue ha